src/core/navbar/navbar-user/navbar-user.component.html

```diff
@@ -0,0 +1,73 @@
+<div *ngIf="!loggedIn()">
+  <div class="buttons has-addons">
+    <a class="button" (click)="authService.login(rememberLogin())">
+      <span class="icon-text">
+        <span class="icon">
+          <ng-icon name="remixLoginBoxLine"></ng-icon>
+        </span>
+        <span translate="core.login"></span>
+      </span>
+    </a>
+    <button
+      class="button has-tooltip-bottom"
+      data-tooltip="Remember Login"
+      id="remember"
+    >
+      <span
+        class="icon"
+        [class.has-text-primary]="rememberLogin()"
+        [class.has-opacity-60]="!rememberLogin()"
+        (click)="toggleRemember()"
+      >
+        <ng-icon size="1.6em" name="remixRotateLockFill"></ng-icon>
+      </span>
+    </button>
+  </div>
+</div>
+
+<div
+  *ngIf="userDetails()"
+  class="dropdown is-right"
+  [class.is-active]="userDropdownActive()"
+  is-auto-hide
+  [is-active-signal]="userDropdownActive"
+>
+  <div class="dropdown-trigger" (click)="userDropdownActive.toggle()">
+    <div
+      id="user-profile-button"
+      class="button"
+      [style.background-image]="userAvatar()"
+    ></div>
+  </div>
+  <div class="dropdown-menu">
+    <div class="dropdown-content">
+      <div class="dropdown-item has-text-centered">
+        <p class="icon-text has-text-weight-bold is-flex-wrap-nowrap">
+          <span
+            *ngIf="userDetails()!.administrator"
+            class="icon has-tooltip-top"
+            data-tooltip="Administrator"
+          >
+            <ng-icon name="remixVipCrown2Fill"></ng-icon>
+          </span>
+          <span style="white-space: nowrap">
+            {{ userDetails()!.name }}
+          </span>
```

Comment:
The administrator crown icon shifts the users full name to the right which looks wrong in comparison to a non-admin user

Reply:
![image](https://github.com/user-attachments/assets/cf3af4f2-3524-45a4-8450-01e0e3148a95)
I think this looks worse

Reply:
Hmm, also true, maybe we can find another way later on (e.g. placing that crown over the profile picture in one of the lower corners)